### ISSUE TITLE
DNS crashes several times per day

### ISSUE THREAD

**Issue by TungstenE2** [pi-hole/pi-hole#4164]
### Versions

  Pi-hole version is v5.3.1 (Latest: v5.3.1)
  AdminLTE version is v5.5 (Latest: v5.5)
  FTL version is v5.8.1 (Latest: v5.8.1)

### Platform

- OS and version:  Raspbian Buster

- Platform:  Raspberry Pi 3b

### Expected behavior

Normal operations

### Actual behavior / bug

In the last couple of weeks I need to restart DNS either by 'pihole restartdns' or via Settings: Restart DNS resolver several times per day. 

Already tried to repair by pihole -r, but this did not do the trick.

Before pihole was working fine for over a year.

### Steps to reproduce

DNS service randomly stopps several times per day.
Need to restart by 'pihole restartdns' or via Settings: Restart DNS resolver several times per day. 

## Debug Token

- URL:   https://tricorder.pi-hole.net/rgy8e9snq4

### Screenshots

![grafik](https://user-images.githubusercontent.com/39955685/117443737-d9151900-af38-11eb-9183-52162c3ebc33.png)
![grafik](https://user-images.githubusercontent.com/39955685/117444000-298c7680-af39-11eb-98ed-334eeec2411b.png)

### Additional context

_Add any other context about the problem here._

**Comment by TungstenE2** [comment]
Thx for your research and feedback.

I deleted the Query database /etc/pihole/pihole-FTL.db and restarted the Pi.

Seem like this fixed the issue partly, but I would have expected that 'pihole -r' would also repair a database corruption. Seems like this is not the case. 

After 18hrs the issue did not reoccure, but I just had the issue that no DNS was resolved, but the dashboard did not show any error. Restart DNS resolver fixed the issue for the moment again.
